### ISSUE TITLE
Strange error on PUT 

### ISSUE THREAD

**Issue by davidascher** [participedia/api#130]
```
error: Exception in PUT /case/953 => error: column "authors" of relation "cases" does not exist
PUT /case/953 500 24.688 ms - 171
```

Haven't dug into what's happening.  It happens when saving an edited case, sometimes.

**Comment by dethe** [comment]
Hmm, it definitely has such a column. I'll see if I can reproduce and fix.

**Comment by dethe** [comment]
When frontend sent a change it didn't send authors, but it got back an authors with an extra entry (because sending a change increments authors by one) and sends authors on the subsequent call. The edit API wasn't filtering out authors (which is a generated field, api-side) and tried to save it, which broke, but not on the first edit call. 

Fixed in PR #134
